vnext/Microsoft.ReactNative/ReactHost/ReactHost.cpp

```diff
@@ -285,10 +285,28 @@ class ReactInspectorHostTargetDelegate : public jsinspector_modern::HostTargetDe
   ReactInspectorHostTargetDelegate(Mso::WeakPtr<ReactHost> &&reactHost) noexcept : m_reactHost(std::move(reactHost)) {}
 
   jsinspector_modern::HostTargetMetadata getMetadata() override {
-    // TODO: (vmoroz) provide more info
-    return {
-        .integrationName = "React Native Windows (Host)",
-    };
+    jsinspector_modern::HostTargetMetadata metadata;
+    metadata.integrationName = "React Native Windows (Host)";
+    metadata.platform = "windows";
+
+    if (Mso::CntPtr<ReactHost> reactHost = m_reactHost.GetStrongPtr()) {
+      auto options = reactHost->Options();
+      if (!options.Identity.empty()) {
+        std::string identity = options.Identity;
+        size_t lastSlash = identity.find_last_of("\\/");
```

Comment:
It would better not to cut the identity name. If some characters are illegal, then it is better to replace them with some legal characters like "-" or "_".